apps/meteor/jest.config.ts

```diff
@@ -32,11 +32,16 @@ export default {
 
```

Comment:
You dont need these for server unit tests

Reply:
The test was not being executed without adding its path. No matching path for the file was present.
Without the adding meteor to module mapper, the test had errors of not being able to resolve meteor/meteor

Reply:
@abhinavkrin, unfortunately, that is a Meteor limitation, but Kevin is correct, this only happens because we are importing the callbacks file directly, we need to import it using proxyquire, otherwise, it will import the whole dep tree which includes meteor/meteor.